src/main/java/emissary/server/mvc/adapters/DirectoryAdapter.java

```diff
@@ -272,14 +272,13 @@ private DirectoryEntryMap zoneTransfer(final String key, @Nullable final String
         }
 
         if (map == null) {
-            throw new EmissaryException("Unable to perform zone transfer to " + key + ": received map is null, isError=" + ws.getStatus()
-                    + ", msgBody=" + ws.getContentString());
+            throw new EmissaryException("Unable to perform zone transfer to " + key + ": received map is null, isError="
+                    + (ws == null ? null : ws.getStatus() + ", msgBody=" + ws.getContentString()));
```

Comment:
"ws.getContentString()" should probably also be protected against ws being null.